chemreg/compound/tests/test_views.py

```diff
@@ -233,6 +231,64 @@ def test_deprecated_qst_in_illdefined(
     )
 
 
+@pytest.mark.django_db
+def test_nonexistent_qst_in_illdefined(
+    admin_user, client, ill_defined_compound_factory, query_structure_type_factory
+):
+    client.force_authenticate(user=admin_user)
+    qst = query_structure_type_factory.create().instance._meta.model
+    nonexistent_pk = 47 if not qst.objects.filter(pk=47).exists() else 74
```

Comment:
This is functional but theoretically could break.  Replacing it with the highest pk + 1 should be unbreakable (unless I'm misunderstanding how pytest runs).

Also that's an abstract way of getting the model class.  I personally prefer fetching it off Django's app registry.

```suggestion
    from django.apps import apps
    from django.db.models import Max
    
    qst = apps.get_model("compound", "QueryStructureType")
    nonexistent_pk = qst.objects.aggregate(Max("pk"))["pk__max"] + 1
```

Reply:
fine with me, I think if we're not going to make one with `.create()` we might be able to assume that the pk of the QST created in the migration is always going to be`1` and just give it any other INT, that's all I was doing with the ternary between `47` and `74` was to assure that the `pk` of the one created here isn't `47`.